public/app/features/integrated-alerting/components/AlertRules/AddAlertRuleModal/AddAlertRuleModal.utils.ts

```diff
@@ -0,0 +1,58 @@
+import { AddAlertRuleFormValues } from './AddAlertRuleModal.types';
+import { AlertRuleCreatePayload, AlertRulesListPayloadFilter } from '../AlertRules.types';
+import { Template } from '../../AlertRuleTemplate/AlertRuleTemplatesTable/AlertRuleTemplatesTable.types';
+import { SelectableValue } from '@grafana/data';
+
+export const formatTemplateOptions = (templates: Template[]): Array<SelectableValue<string>> =>
+  templates.map(template => ({
+    value: template.summary,
+    label: template.summary,
+  }));
+
+// TODO: handle new types as they gets added to AlertRuleFilterType
+export const formatFilter = (filter: string): AlertRulesListPayloadFilter => {
+  if (!filter) {
+    return {
+      key: '',
+      value: '',
+      type: 'EQUAL',
+    };
+  }
+
+  const [key, value] = filter.split('=');
+
+  return {
+    key,
+    value,
+    type: 'EQUAL',
```

Comment:
If there will be more types in the future maybe it's better to use an enum.

Reply:
There is an enum already, that's a fixed value for it, I'd leave it as it is, since it has to be updated anyway in that case